packages/xo-web/src/common/xo/index.js

```diff
@@ -345,9 +345,8 @@ export const subscribeRoles = createSubscription(
 
 export const subscribeIpPools = createSubscription(() => _call('ipPool.getAll'))
 
-export const subscribeResourceCatalog = createSubscription(() =>
-  _call('cloud.getResourceCatalog')
-)
+export const subscribeResourceCatalog = ({ filters } = {}) =>
```

Comment:
Dynamically creating subscriptions is a bad idea and must be done very carefully, take a look at this code for instance: https://github.com/vatesfr/xen-orchestra/blob/47819ea95620880579111ca9d25bbfda8ec82acc/packages/xo-web/src/common/xo/index.js#L406-L430

I think you'd better create a new `subscribeHubResourceCatalog` subscription.